remix-simulator/src/genesis.js

```diff
@@ -0,0 +1,40 @@
+var EthJSBlock = require('ethereumjs-block')
+var RemixLib = require('remix-lib')
+var executionContext = RemixLib.execution.executionContext
+var ethJSUtil = require('ethereumjs-util')
+var BN = ethJSUtil.BN
+
+function checkpointAndCommit (cb) {
+  if (executionContext.vm().stateManager._checkpointCount > 0) {
```

Comment:
I'd rather ask the ethereumjs-vm dev before using an `_` private property. I recall i was doing the same for an another issue, that was breaking some other part of the code...

Reply:
look like duplicated with https://github.com/ethereum/remix/pull/1211/files#diff-e03a02494c49f81bf77f43f457e5fc39R158